astropy/wcs/src/wcslib_tabprm_wrap.c

```diff
@@ -413,7 +414,7 @@ PyTypeObject PyTabprmType = {
   0,                            /*tp_getattr*/
   0,                            /*tp_setattr*/
   0,                            /*tp_compare*/
-  (reprfunc)PyTabprm___str__,   /*tp_repr*/
```

Comment:
What is the reason for removing the repr? 

If that wasn't an accidental removal we probably should also remove the function implementation of `PyTabprm___str__`

Reply:
I removed `__repr__` because I hated the output of it when I was debugging my code. However, it is really a matter of taste and whether people prefer excessive vs succinct output. To understand (=see for yourself) what I am talking about, try: 

```python
>>> from astropy import wcs
>>> 5 * [wcs.WCS().wcs]
```

and now please find the `cdelt` of the 2nd `WCS` in the printed output. So, for me it was easier to see how many `tabprm` or `wtbarr` structures I had in the members `WCS.wcs.tab` and `WCS.wcs.wtb`. If I wanted to see the content of one particular structure, I would do `print(WCS.wcs.wtb[0])`.

Maybe my above example is not appropriate for a single WCS object (and that is how it is used most of the time), but for lists of `tabprm` and `wtbarr` structures, it felt overwhelming. Maybe a more compact format can be found for `__repr__` that is also more informative than a just object address.